aws/resource_aws_ecs_service_test.go

```diff
@@ -10,6 +10,7 @@ import (
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/aws/aws-sdk-go/service/servicediscovery"
```

Comment:
Had to remind myself that this is a `*_test.go` file so `semgrep` automatically ignores the multiple AWS Go SDK service imports 👍